src/components/message/message-card/MessageCardTools.tsx

```diff
@@ -64,10 +64,10 @@ const MessageCardTools = ({
 
 	const viewTypes = message.body
 		? [
-				MessageViewType.JSON,
 				MessageViewType.FORMATTED,
```

Comment:
why has order changed? i think json and formatted view types should be next to each other

Reply:
according to https://www.figma.com/file/vB9EyLv1GBE3vJqy2Hxk08/TH2-New-Report-Style-Updates?node-id=2738%3A85943

Reply:
i think previous order makes more sense. let's leave it as it was